src/Foundatio.Storage.SshNet/Storage/SshNetFileStorage.cs

```diff
@@ -249,7 +249,7 @@ public SshNetFileStorage(Builder<SshNetFileStorageOptionsBuilder, SshNetFileStor
 
             string[] userParts = uri.UserInfo.Split(new [] { ':' }, StringSplitOptions.RemoveEmptyEntries);
             string username = Uri.UnescapeDataString(userParts.First());
```

Comment:
I'm wondering if we should also check the length here or do `userParts.FirstOrDefault() ?? String.Empty`